jbi/services/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Exports the bugzilla and jira services modules for convenience"""
+
+from . import bugzilla, jira
```

Comment:
You almost could have left `get_jira` and `get_bugzilla` here.

```py
_BZ_CLIENT = None

def get_bugzilla(*args, **kwargs):
    global _CLIENT  # pylint: disable=global-statement
    if not _BZ_CLIENT:
        _BZ_CLIENT = bugzilla.get_client()
    return _BZ_CLIENT
```

(Just a drive-by idea, not a change request)

Reply:
Yeah, I kept it this way for now since we have `<service module>.get_client` and `<service module>.other_function`. If we end up putting service access through one class, then getters in `init` probably make more sense